.github/workflows/auto-release-rc.yml

```diff
@@ -0,0 +1,95 @@
+# Compares the version in pyproject.toml to tags on the repo. If the tag doesn't exist, a new tag is created, which
+# then triggers the normal "on tag" release automation in the build job
+name: Auto Tag RC
+
+on:
+  push:
+    branches:
+      - develop
+      - tna-in-releases2
+
+concurrency:
+  group: rc-release-check
+
+jobs:
+  check-version:
+    name: Check version increment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clean workspace
+        uses: Chia-Network/actions/clean-workspace@main
+
+      - name: Checkout current branch
+        uses: actions/checkout@v4
+        with:
+          # Need PACKAGE_ADMIN_PAT token so when the tag is created, the tag automation runs
+          token: ${{ secrets.PACKAGE_ADMIN_PAT }}
+          fetch-depth: 0
+
+      # Install Python for commitizen, the automatic changelog tool
+      - name: Setup Python
+        uses: Chia-Network/actions/setup-python@main
+        with:
+          python-version: "3"
```

Comment:
is 3.13, etc all ok here? or should this be pinned to like.. 3.12 or something? (I dont think 3.13 works w/ chia yet)